builders/tools/sidecar.md

```diff
@@ -0,0 +1,153 @@
+
+```
+node_modules/.bin/substrate-api-sidecar --version
+```
+
+## Setting up the Substrate API Sidecar {: #setting-up-the-substrate-api-sidecar }
+
+In the terminal that Sidecar will run, export the environmental variable for the WS endpoint of the network. Examples: 
+
+=== "Moonriver"
+    ```
+    export SAS_SUBSTRATE_WS_URL=wss://wss.moonriver.moonbeam.network
+    ```
+
+=== "Moonbase Alpha"
+    ```
+    export SAS_SUBSTRATE_WS_URL=wss://wss.testnet.moonbeam.network
+    ```
+
+=== "Moonbeam Dev Node (Default)"
+    ```
+    export SAS_SUBSTRATE_WS_URL=ws://127.0.0.1:9944
+    ```
+
+Please reference the [Public Endpoints](/builders/get-started/endpoints/) page for a full list of Moonbeam network endpoints.
+
+After setting the environmental variable, you can use the `echo` command to check that the environmental variable has been set correctly, by typing:
+
+```
+echo $SAS_SUBSTRATE_WS_URL
+```
+
+And it should display the network endpoint you have just set. 
+
+## Running Substrate API Sidecar {: #running-substrate-api-sidecar } 
+
+With the network endpoint environmental variable set, and from the installation directory root, run:
+
+```
+node_modules/.bin/substrate-api-sidecar 
+```
+
+If the installation and configuration are successful, you should see this output in the console: 
+
+![Successful Output](/images/builders/tools/sidecar/sidecar-1.png)
+
+## Substrate API Sidecar Endpoints {: #substrate-api-sidecar-endpoints } 
+
+Some of the commonly used Substrate API Sidecar endpoints include:
+
+ - **GET /blocks​/head** — Get the most recently finalized block. The optional parameter `finalized` can be set to `false` to the get the newest known block, which may not be finalized.
+ - **GET /blocks/head/header** — Get the most recently finalized block header. The optional parameter `finalized` can be set to `false` to the get the newest known block header, which may not be finalized. 
+ - **GET /blocks/{blockId}** — Get a block by its height or hash.
+ - **GET /accounts/{accountId}/balance-info** — Get balance information for an account.
+ - **GET /node/version** — Get information about the Substrates node's implementation and versioning.
+ - **GET /runtime/metadata** — Get the runtime metadata in decoded, JSON form.
+
+For a full list of API endpoints available on Substrate API Sidecar, please refer to the [official documentation](https://paritytech.github.io/substrate-api-sidecar/dist/).
+
+## EVM Field Mapping in Block JSON Object {: #evm-fields-mapping-in-block-json-object}
+
+Substrate API Sidecar returns Moonbeam blocks as a JSON object. Information related to EVM execution of Moonbeam transactions is under the `extrinsics` top level field, where individual extrinsics are organized numerically as nested JSON objects. 
+
+Moonbeam EVM transactions can be identify by the `method` field under the current extrinsic object, where it is set to:
+
+```
+pallet: "ethereum", method: "transact" 
```

Comment:
I think it would be nice to show how is the nesting

```
RESPONSE
  |--extrinsicts
          |--method
                  |--pallet: "ethereum"
                  |--method: "transact"
          |--signature:
          |--nonce: 
          |--args:
          
          
        I think it is important also to say that elements of the regular Ethereum transaction object are under `args`.

Reply:
You could also do (simpler) `RESPONSE.extrinsics.method.pallet = "ethereum"` and `RESPONSE.extrinsics.method.method = transact`

Not sure what do you prefer